index.js

```diff
@@ -296,11 +296,11 @@ class KeyringController extends EventEmitter {
   // This method signs tx and returns a promise for
   // TX Manager to update the state after signing
 
-  signTransaction (ethTx, _fromAddress) {
+  signTransaction (ethTx, _fromAddress, opts) {
```

Comment:
I'd just like us to keep new params like `opts` optional across the board if possible.